_events/2023/2023-05-diverse-horsfall.md

```diff
@@ -0,0 +1,30 @@
+---
+title: The importance of talking about mental health in the Research Software Engineering community
+subtitle: DiveRSE Seminar
+expires: 2023-05-17
+event_date: "May 16, 2023"
+layout: event
+duration: 60
+repeated: false
+category: dei
+time:
+    - - start: 2023-05-16T09:00:00Z
+        end: 2023-05-16T10:00:00Z
+---
+
+In the first of our 2023 series of DiveRSE talks, Dave Horsfall (University of Newcastle and Software
+Sustainability Institute Fellow) will talk about his work advocating for better mental health within the
+Research Software Community.
+
+[**Event Website**](https://diverse-rse.github.io/events/2023-05-16)
+
+The talk has been timed to coincide with [Mental Health Awareness Week](https://www.mentalhealth.org.uk/our-work/public-engagement/mental-health-awareness-week-about) which is taking place in the
+UK from the 15th-21st May.
+
+You can find out more about Dave’s work on mental health advocacy at: [https://horsfall.dev/mental-health/](https://horsfall.dev/mental-health/])
```

Comment:
There's a stray closing bracket in the URL.

Reply:
Yes there is! Whoops! Lemme fix that, thanks!